bool request
---
uint8 pan
uint8 tilt